packages/components/src/placeholder/style.scss

```diff
@@ -15,9 +15,11 @@
 	}
 
 	// Block UI appearance.
+	@include block-ui-emphasized();
```

Comment:
Using "block-ui-emphasized" in placeholder implies that placeholder is specific to blocks and the fact that it's on the components package suggests that it's not.

1- I'd rename the mixin
2- I'd accept the fact that there's some duplication. Abstraction is not always good https://overreacted.io/goodbye-clean-code/

Reply:
👍 👍 I'll revert this bit.